app/controllers/users/mfa_selection_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Users
+    class MfaSelectionController < ApplicationController
```

Comment:
this looks like it's indented too far (and the rest of the class too)
```suggestion
  class MfaSelectionController < ApplicationController
```